damus/Views/Profile/EditMetadataView.swift

```diff
@@ -103,13 +103,15 @@ struct EditMetadataView: View {
             TopSection
             Form {
                 Section(NSLocalizedString("Your Name", comment: "Label for Your Name section of user profile form.")) {
-                    TextField("Satoshi Nakamoto", text: $display_name)
+                    let display_name_placeholder = "Satoshi Nakamoto"
+                    TextField(display_name_placeholder, text: $display_name)
                         .autocorrectionDisabled(true)
                         .textInputAutocapitalization(.never)
                 }
                 
                 Section(NSLocalizedString("Username", comment: "Label for Username section of user profile form.")) {
-                    TextField("satoshi", text: $name)
+                    let username_placeholder = "satoshi"
```

Comment:
Extracting it out like this so that it doesn't get flagged as something that needs to be translated.